config/environments/development.rb

```diff
@@ -27,6 +27,9 @@
   # Raise an error on page load if there are pending migrations
   config.active_record.migration_error = :page_load
 
+  # Better logging for docker-compose
+  config.logger = Logger.new(STDOUT)
```

Comment:
yeah, I think this should become standard, I was doing something similar on feeder, but this is a better way.